app/src/main/java/com/kickstarter/viewmodels/MessageThreadsViewModel.java

```diff
@@ -151,14 +187,14 @@ public ViewModel(final @NonNull Environment environment) {
     private final PublishSubject<Void> onResume = PublishSubject.create();
     private final PublishSubject<Void> swipeRefresh = PublishSubject.create();
 
-    private final Observable<Boolean> hasNoMessages;
-    private final Observable<Boolean> hasNoUnreadMessages;
+    private final BehaviorSubject<Boolean> hasNoMessages = BehaviorSubject.create();
+    private final BehaviorSubject<Boolean> hasNoUnreadMessages = BehaviorSubject.create();
     private final BehaviorSubject<Boolean> isFetchingMessageThreads = BehaviorSubject.create();
     private final BehaviorSubject<List<MessageThread>> messageThreadList = BehaviorSubject.create();
-    private final Observable<Integer> unreadCountTextViewColorInt;
-    private final Observable<Integer> unreadCountTextViewTypefaceInt;
+    private final BehaviorSubject<Integer> unreadCountTextViewColorInt = BehaviorSubject.create();
+    private final BehaviorSubject<Integer> unreadCountTextViewTypefaceInt = BehaviorSubject.create();
     private final Observable<Boolean> unreadCountToolbarTextViewIsGone;
-    private final Observable<Integer> unreadMessagesCount;
+    private final BehaviorSubject<Integer> unreadMessagesCount = BehaviorSubject.create();
```

Comment:
these all became `BehaviorSubject`s cause our `unreadMessagesCount` now also relies on a `project` from the `intent()`

Reply:
cool.